locksmith/migrations/20190205161953-create_transactions.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Transactions', {
+      transactionHash: {
+        type: Sequelize.STRING,
+        primaryKey: true
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      sender: {
+        allowNull: false,
```

Comment:
We probably want to add indices on several fields, right? On top of my head I am thinking `createdAt`, `sender`, `recipient`, but maybe there are other?

Reply:
I think this will be a good idea, but would like to ensure that we are adding those based on the use cases discussed. How does indexing by sender, and (sender/created_at) sound for now?

Reply:
Yes! Good!